elements/lisk-chain/src/data_access/constants.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export const DB_KEY_BLOCKS_ID = 'blocks:id:';
+export const DB_KEY_BLOCKS_HEIGHT = 'blocks:height:';
+export const DB_KEY_TRANSACTIONS_BLOCK_ID = 'transactions:blockID:';
+export const DB_KEY_TRANSACTIONS_ID = 'transactions:id:';
+export const DB_KEY_TEMPBLOCKS_HEIGHT = 'tempBlocks:height:';
+export const DB_KEY_ACCOUNTS_ADDRESS = 'accounts:address:';
+export const DB_KEY_CHAIN_STATE = 'chain:';
+export const DB_KEY_CONSENSUS_STATE = 'consensus:';
```

Comment:
Please remove the `:` suffix from key names. Eventually, the key will be calculated with the additional suffix e.g. block id or height, so add that `:` in there. 

Or other solution would convert these to a template function which takes the object of concerning data and generate the full key.

```js
export const DB_KEY_BLOCKS_ID = ({id}) => `blocks:id:${id}`;
```

Reply:
we will remove the suffix `:` from the constants